operations/research-and-development/README.md

```diff
@@ -1,20 +1,48 @@
-* [WIP: Feature Idea Flow Chart](https://docs.google.com/drawings/d/1D6KiN31mhNr1A0DGGHOuu6hvS0gpDTFuftnh2yyniNQ/edit): Work-in-progress
 
 **Product Strategy**
 
 * [Analyst Research](https://community.mattermost.com/private-core/channels/analyst-research): Analyst meeting tracker, briefing procedures, research. We are currently Gartner clients.
 * [Compete](https://community.mattermost.com/private-core/channels/compete): Key articles on competitors. Also see [automated feeds from competitor marketing](https://community.mattermost.com/private-core/channels/compete-feeds).
 
-## Leadership
+### What tools we use
+
+We use a variety of tools at Mattermost. Not every team uses the same tools, and it can get overwhelming to try remember who uses what and where it is.
+
+During onboarding you'll be provided with a list of tools to set up. You can also visit the [Helpdesk](https://helpdesk.mattermost.com/support/catalog/items) to find the tool and request access.
+
+### Where to find information
+
+We use a range of web properties and tools to document and share plans, specs, roadmaps, and release schedules.
+
+* Product roadmaps, feature releases, planned features: [Features by release productboard](https://mattermost.productboard.com/roadmap/2855466-features-by-release), [Release roadmap by vertical](https://mattermost.productboard.com/roadmap/2941622-release-roadmap-by-product-vertical).
+* Product specs: [Teams](https://mattermost.atlassian.net/wiki/spaces?label=teams), [Software project](https://mattermost.atlassian.net/wiki/spaces?label=software-project)
+* Apps, integrations, plugins: [Integrate](https://developers.mattermost.com/integrate/getting-started/)
+* How to get started with contributing and developer set up: [Getting started](https://developers.mattermost.com/contribute/getting-started/)
+* Designs and mockups: [Figma](https://www.figma.com/files/802235376456811310/recent?fuid=788840938074139976)
+* How open source software is developed: [Development & Release Process](https://handbook.mattermost.com/operations/research-and-development/product/release-process/release-overview)
+* How contributors can get involved: [Platform Contribution Process](https://handbook.mattermost.com/contributors/contributors/community-playbook)
+* Analytics playbook and data wallows: [Analytics](https://community.mattermost.com/private-core/channels/analytics-2)
+* Work in progress: [WIP: Feature Idea Flow Chart](https://docs.google.com/drawings/d/1D6KiN31mhNr1A0DGGHOuu6hvS0gpDTFuftnh2yyniNQ/edit)
+
+### Where to find us
+
+* Technical writers: [DWG: Documentation working group](https://community.mattermost.com/private-core/channels/dwg-documentation-working-group)
+* Product Managers: [Product Management](https://community.mattermost.com/private-core/channels/product-management)
+* UX team: [UX Design](https://community-release.mattermost.com/private-core/channels/ux-design-private)
+
```

Comment:
Also add

* Analysts: [Analytics](https://community.mattermost.com/private-core/channels/analytics-2)
* Community: [Community Team](https://community.mattermost.com/private-core/channels/community-team)

Reply:
^as an aside, I wonder which of these channels could be moved to the Contributors team, given most of the conversations aren't confidential.

Reply:
Great point. I've changed the UX design on to the Contributors team in the interim.